UMLEditor/UMLEditor/Views/ClassBox.axaml.cs

```diff
@@ -169,7 +172,8 @@ public ClassBox()
                 // Set the ClassBox's location to the location of the cursor
                 Canvas.SetLeft(this, newX);
                 Canvas.SetTop(this, newY);
-
+                
+                /// ****** COMMENT OUT THESE LINES FOR BETTER DRAW PERFORMANCE ****** 
                 _parentWindow!.RedrawLines();
```

Comment:
The performance hit for real time updates is far too expensive and makes the editor unusable. Killian said it was fine to not have real time updates, so I think we should go this route. To do this, you should only comment out the call to RedrawLines. ReconsiderCanvasSize needs to be called so the canvas can grow when a classbox is moved over the edge of the screen.